public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilterItem.tsx

```diff
@@ -25,7 +25,7 @@ export function LabelFilterItem({ item, defaultOp, onChange, onDelete, onGetLabe
   }>({});
 
   const isMultiSelect = () => {
-    return item.op === operators[0].label;
+    return item.op === operators[0].label || item.op === operators[3].label;
```

Comment:
nitpick: this is a little hard to read (which are the zero-element and the three-element?).. maybe we could modify the `operators` structure, to be like:

```ts
const operators = [
  { label: '=~', value: '=~' , isMultiselect: true},
.
.
.
];
```

?

Reply:
Yeah, I was actually thinking about this as well. Good call. I'll change it.

Reply:
Updated in https://github.com/grafana/grafana/pull/52978/commits/d7191be3492a3b90a69ab7d3f09356202fad1136